cli/crates/backend/src/api/graphql/types/mutations/environment_variable_upsert_by_slugs.rs

```diff
@@ -28,13 +28,15 @@ pub struct NameContainsInvalidCharactersError {
 #[derive(cynic::QueryFragment, Debug)]
 #[cynic(graphql_type = "Mutation", variables = "EnvironmentVariableUpsertBySlugsVariables")]
 pub struct EnvironmentVariableUpsertBySlugs {
+    #[allow(dead_code)]
```

Comment:
Can't we delete the dead code?

Reply:
Another PR. There's lots of it, so just removed the things Yoav added in this PR.